src/Microsoft.AspNet.Mvc.TagHelpers/TagHelperOutputExtensions.cs

```diff
@@ -100,5 +106,70 @@ public static class TagHelperOutputExtensions
                 tagHelperOutput.Attributes.Remove(attribute);
             }
         }
+
+        private static void CopyHtmlAttributeMaintainOrder(
+            int allAttributeIndex,
+            TagHelperOutput tagHelperOutput,
+            TagHelperContext context)
```

Comment:
Nah, other `CopyHtmlAttribute` method is an extension. Otherwise it wouldn't have same order.